### ISSUE TITLE
Anführungszeichen

### ISSUE THREAD

**Issue by zuphilip** [Informationspraxis/artikel-vorlage#19]
Aktuell werden PDF und HTML die US-amerikanische Variante der Anführungszeichen verwendet: “…”. Im Deutschen würde man aber stattdessen die "9966"er Anfürhungszeichen unten dann oben verwenden, d.h. „…“. (Die Schweiz weicht davon teilweise ab und verwendet dann stattdessen eher eine Variante der französischen Anführungszeichen: «…», aber nur in einigen Dokumenten, vgl. https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Deutschland_und_%C3%96sterreich .

Wollen wir in Zukunft uns möglichst konsequent an die Deutsch Variant „…“ halten?

Dazu müsste man wohl noch etwas in der LaTeX-Vorlage und bei den Einstellungen des `csquote` Paketes ändern für die PDF-Fahne sowie den HTML-Workflow genauer betrachten (ePUB habe ich jetzt einfach mal aussen vor gelassen).

**Comment by zuphilip** [comment]
Die falsche Konversion scheint schon im ersten Schritt nach Markdown zu erfolgen: In Word sind die korrekten Anfürhungszeichen `„…“` noch enthalten aber in Markdown ist dann das zweite davon ersetzt worden. d.h. insgesamt zu `„…"`. Verantwortlich scheint dafür die Pandoc-Option [smart](https://pandoc.org/MANUAL.html#extension-smart) zu sein. Wenn man diese Option für die Konversation in Markdown deaktiviert, dann funktioniert die Konversation wie gewünscht. Unerwünschte Nebeneffekte dabei sollten nicht auftreten, da sowohl HTML wie auch die LaTeX-Vorlage mit den Unicode-Zeichen für die Anführungszeichen und Dashes umgehen können.

**Comment by hauschke** [comment]
Das bedeutet, wir können die negativen Effekte durch eine pandoc-Option umgehen, was wir dann nur dokumentieren müssten. Sehr gut, danke!

**Comment by zuphilip** [comment]
Ja, genau. Wenn man die "smart" Option beim ersten Schritt mit pandoc deaktiviert, dann klappt es mit den richtig gesetzen Anführungszeichen. Dies habe ich jetzt bereits im #13 umgesetzt, falls Du @hauschke Dir das ansehen willst oder gleich ausprobieren und annehmen...
